app/charts/column/columns-grouped-state.tsx

```diff
@@ -287,9 +263,10 @@ const useGroupedColumnsState = (
     const xScaleIn = scaleBand().domain(segments).padding(PADDING_WITHIN);
 
     // x as time, needs to be memoized!
-    const xEntireDomainAsTime = extent(plottableSortedData, (d) =>
-      getXAsDate(d)
-    ) as [Date, Date];
+    const xEntireDomainAsTime = extent(sortedData, (d) => getXAsDate(d)) as [
+      Date,
+      Date
+    ];
```

Comment:
Why `sortedData` and not `plottableSortedData` as before? The same comment for lines 308 and 446 😅

Reply:
No reason, thanks for noticing.